client/my-sites/plans/jetpack-plans/faq/index.tsx

```diff
@@ -62,10 +79,21 @@ const JetpackFAQ: React.FC = () => {
 						}
 					) }
 				</FoldableFAQ>
+				<FoldableFAQ
+					id="backup-storage-limits"
+					question={ translate( 'How do backup storage limits work?' ) }
+					onToggle={ onFaqToggle }
+					className="jetpack-faq__section"
+				>
+					{ translate(
+						"If your site's Backup storage limit is reached, your older backups will be deleted. Depending on the size of your site and your site's Backup storage limit, your site's backup retention period may be reduced down to 7 days of your most recent backups. You will still be able to restore existing backups, but new site updates will not be backed up until you free up storage or upgrade your storage limit here"
```

Comment:
Nice catch, I think the "here" wasn't present in the initial mockup but got added later on, and then I just copied and pasted. I'll link this to the https://cloud.jetpack.com/pricing/storage/:site url (Thats probably best right?)

Reply:
Actually unless they have the `:site` in the URL, this would just send them to the Pricing page. I'll have to see if there's a better place to link them to

Reply:
> I'll link this to the https://cloud.jetpack.com/pricing/storage/:site url (Thats probably best right?)

Yes, that was my first thought...  But now I just realized that in some scenarios, there is no site in context, like when visiting `cloud.jetpack.com/pricing`.  So I would say we either remove the word "here", or another option might be to add the "here" link only if the site is in context, otherwise leave it out. So whatever you think, it's your call. 🙂

Reply:
I consulted the team and we decided to remove it :)